iconservice/iconscore/icon_score_context.py

```diff
@@ -101,35 +101,21 @@ class IconScoreContext(object):
     """Contains the useful information to process user's jsonrpc request
     """
 
-    def __init__(self,
-                 context_type: 'IconScoreContextType' = IconScoreContextType.QUERY,
-                 func_type: 'IconScoreFuncType' = IconScoreFuncType.WRITABLE,
-                 block: 'Block' = None,
-                 tx: 'Transaction' = None,
-                 msg: 'Message' = None,
-                 block_batch: 'BlockBatch' = None,
-                 tx_batch: 'TransactionBatch' = None,
-                 new_icon_score_mapper: 'IconScoreMapper' = None) -> None:
+    def __init__(self, context_type: 'IconScoreContextType' = IconScoreContextType.QUERY) -> None:
```

Comment:
I removed parameters except for context_type because when initializing IconScoreContext in the area that uses the IconScoreContext, do not pass any other parameters.